backend/infrahub/menu/menu.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import TYPE_CHECKING
 
-from infrahub.core.constants import infrahubkind as InfrahubKind
+from infrahub.core.constants import infrahubkind as InfrahubKind  # noqa: N812
```

Comment:
I think on this one and in the conflict_recorder the import should instead be as below like we have everywhere else:

```python
from infrahub.core.constants import InfrahubKind
```

Not sure if there's a reason why it was like this from the start, but probably just a minor error.